### PR TITLE
artifactory key update

### PR DESCRIPTION
**Pull request by garyverhaegen-da**

### CODE REVIEW

File: docs/2.6.5/docs/getting-started/installation.rst [hunk 1]
```diff
@@ -106,18 +106,23 @@ can install it as follows:
...
+  <https://digitalasset.jfrog.io/ui/user_profile>`_.
 
 .. code-block:: yaml
 
    artifactory-api-key: YOUR_API_KEY
 
-This will be used by the assistant to download other versions automatically from artifactory.
+This will be used by the assistant to download other versions automatically
+from Artifactory. The config key is called `artifactory-api-key` for historical
+reasons, but will work just as well with an Identity Token as with an API Key.
```
Comment (marciw-da) [inline docs/2.6.5/docs/getting-started/installation.rst:125]:
```suggestion
reasons, but you can use either an identity token or an API key.
```

File: docs/2.7.4/docs/getting-started/installation.rst [hunk 2]
```diff
@@ -108,18 +108,23 @@ can install it as follows:
...
-  Artifactory user profile.
+  releases.
+- On Linux and MacOS, download the corresponding tarball, extract it and run
+  ``./install.sh``. Afterwards, modify the :ref:`global daml-config.yaml
+  <global_daml_config>` and add an entry with your Artifactory token. The Daml
+  Enterprise tooling will accept either an Artifactory API Key or an
+  Artifactory Identity Token, though Artifactory API Keys have been deprecated
+  by Artifactory and may stop working after the end of 2023. Both API Keys and
+  Identity Tokens can be managed from `your Artifactory profile page
+  <https://digitalasset.jfrog.io/ui/user_profile>`_.
```
Comment (marciw-da) [inline docs/2.7.4/docs/getting-started/installation.rst:119]:
```suggestion
- On Linux and MacOS, download the corresponding tarball, extract it, and run
  ``./install.sh``. Afterward, modify the :ref:`global daml-config.yaml
  <global_daml_config>` to add an entry for your Artifactory token. The Daml
  Enterprise tooling accepts either an Artifactory API key or an
  Artifactory identity token, but API keys are deprecated
  by Artifactory as of the end of 2023. To manage API keys and
  identity tokens, go to your `Artifactory profile page
  <https://digitalasset.jfrog.io/ui/user_profile>`_.
```

File: docs/2.7.4/docs/getting-started/installation.rst [hunk 3]
```diff
@@ -108,18 +108,23 @@ can install it as follows:
...
 
 .. code-block:: yaml
 
    artifactory-api-key: YOUR_API_KEY
 
-This will be used by the assistant to download other versions automatically from artifactory.
+This will be used by the assistant to download other versions automatically
+from Artifactory. The config key is called `artifactory-api-key` for historical
+reasons, but will work just as well with an Identity Token as with an API Key.
```
Comment (marciw-da) [inline docs/2.7.4/docs/getting-started/installation.rst:127]:
```suggestion
reasons, but you can use either an identity token or an API key.
```

File: docs/2.8.0/docs/getting-started/installation.rst [hunk 4]
```diff
@@ -108,18 +108,23 @@ can install it as follows:
...
-  Artifactory user profile.
+  releases.
+- On Linux and MacOS, download the corresponding tarball, extract it and run
+  ``./install.sh``. Afterwards, modify the :ref:`global daml-config.yaml
+  <global_daml_config>` and add an entry with your Artifactory token. The Daml
+  Enterprise tooling will accept either an Artifactory API Key or an
+  Artifactory Identity Token, though Artifactory API Keys have been deprecated
+  by Artifactory and may stop working after the end of 2023. Both API Keys and
+  Identity Tokens can be managed from `your Artifactory profile page
+  <https://digitalasset.jfrog.io/ui/user_profile>`_.
```
Comment (marciw-da) [inline docs/2.8.0/docs/getting-started/installation.rst:119]:
```suggestion
- On Linux and MacOS, download the corresponding tarball, extract it, and run
  ``./install.sh``. Afterward, modify the :ref:`global daml-config.yaml
  <global_daml_config>` to add an entry for your Artifactory token. The Daml
  Enterprise tooling accepts either an Artifactory API key or an
  Artifactory identity token, but API keys are deprecated
  by Artifactory as of the end of 2023. To manage API keys and
  identity tokens, go to your `Artifactory profile page
  <https://digitalasset.jfrog.io/ui/user_profile>`_.
```

File: docs/2.8.0/docs/getting-started/installation.rst [hunk 5]
```diff
@@ -108,18 +108,23 @@ can install it as follows:
...
 
 .. code-block:: yaml
 
    artifactory-api-key: YOUR_API_KEY
 
-This will be used by the assistant to download other versions automatically from artifactory.
+This will be used by the assistant to download other versions automatically
+from Artifactory. The config key is called `artifactory-api-key` for historical
+reasons, but will work just as well with an Identity Token as with an API Key.
```
Comment (marciw-da) [inline docs/2.8.0/docs/getting-started/installation.rst:127]:
```suggestion
reasons, but you can use either an identity token or an API key.
```

**Review by marciw-da — COMMENTED** [review]
Suggestions for simplicity and style. Not approving because auto-merge is on; will approve after you've had a chance to review and apply.

**Comment by garyverhaegen-da** [comment]
> Suggestions for simplicity and style. Not approving because auto-merge is on; will approve after you've had a chance to review and apply.

Thank you for the review, and for going the extra mile in making the same edits three times.

**Review by marciw-da — APPROVED** [review]
Thanks! LGTM.

**Pull request merged**